README.md

```diff
@@ -1,57 +1,32 @@
 
-<p align="center">
+<br />
+<br />
 
-# Composable Node
-  <img alt="Composable Finance" title="Composable Finance" src="composable.png">
+<p align="center">
+  <img alt="Composable Finance" title="Composable Finance" src="banner.png">
 </p>
 
+<br />
+<br />
 
-[![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/composablefi/composable)](https://github.com/composablefi/composable/tags) [![Twitter](https://img.shields.io/badge/Twitter-gray?logo=twitter)](https://twitter.com/ComposableFin) [![Discord](https://img.shields.io/badge/Discord-gray?logo=discord)](https://discord.gg/pFZn2GCn65) [![Telegram](https://img.shields.io/badge/Telegram-gray?logo=telegram)](https://t.me/ComposableFinanceAnnouncements) [![Medium](https://img.shields.io/badge/Medium-gray?logo=medium)](https://composablefi.medium.com/)
-[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/45659)
-
-
-Picasso is our custom built Kusama parachain, based on the substrate framework.
-
-## Documentation
-
-To learn more about our ecosystem, vision, and product specifics - visit our 
-[mdbook](https://docs.composable.finance/).
-
-You can also find our cargo docs [here](https://dali.devnets.composablefinance.ninja/doc).
+## Monorepo for Composable Finance
 
-## Install
+[![Latest Release](https://img.shields.io/github/v/tag/composablefi/composable)][latest-url]
+![Build][build-badge]
+[![Discord][discord-badge]][discord-url]
+[![Bors Enabled][bors-badge]][bors-url]
 
-For linux, FreeBSD, OpenBSD and macOS:
+[latest-url]: https://github.com/composablefi/composable/tags
 
-```sh
-rustup update
-rustup update nightly
-rustup target add wasm32-unknown-unknown --toolchain nightly
-git clone https://github.com/composableFi/composable
-cd composable/
-sh scripts/init.sh
-cargo build --release
-```
+[build-badge]: https://github.com/composablefi/composable/actions/workflows/check.yml/badge.svg
 
-or you can simply install it with this one-liner:
+[discord-badge]: https://img.shields.io/badge/Discord-gray?logo=discord
+[discord-url]: https://discord.gg/pFZn2GCn65
 
-```sh
-curl https://get.composable.finance | sh
-```
+[bors-badge]: https://bors.tech/images/badge_small.svg
+[bors-url]: https://app.bors.tech/repositories/45659
 
+## Documentation
 
-## Run  
-
-You may run [Composable parachain with relay chain](scripts/polkadot-launch).
-
-Or via `nix develop .#devnet`.
```

Comment:
assumption we will bring nix soon to run stuff back into dev docs

Reply:
Yeah, just removing that from the README now until we have changed to nix altogether.